test/NonSilo.Tests/OrleansRuntime/AsyncSerialExecutorTests.cs

```diff
@@ -2,23 +2,25 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
 using Orleans;
 using Orleans.Runtime;
 using Orleans.Runtime.Configuration;
+using Orleans.TestingHost.Utils;
 using Xunit;
 
 namespace UnitTests.OrleansRuntime
 {
     public class AsyncSerialExecutorTests
     {
-        public Logger logger;
+        public ILogger logger;
         private SafeRandom random;
         public int operationsInProgress;
 
         public AsyncSerialExecutorTests()
         {
-            LogManager.Initialize(new NodeConfiguration());
-            logger = LogManager.GetLogger("AsyncSerialExecutorTests", LoggerType.Application);
+            var loggerFactory = TestingUtils.CreateDefaultLoggerFactory(new NodeConfiguration().TraceFileName);
+            logger = loggerFactory.CreateLogger<AsyncSerialExecutorTests>();
```

Comment:
just use ITestOutputHelper instead of the logger, which is a lot more appropriate (this logger usage is probably from before we switched to xUnit). Otherwise you'd have to implement IDIsposable and get rid of the new factory, which is a lot more cumbersome.